packages/frontend/component/src/ui/input/input.tsx

```diff
@@ -26,6 +27,7 @@ export type InputProps = {
   endFix?: ReactNode;
   type?: HTMLInputElement['type'];
   inputStyle?: CSSProperties;
+  native?: boolean;
```

Comment:
Write some comments for this prop, I'm confused about why we need this?

Reply:
Because I don't want to handle the Composition event for each` <input />`tag separately, and some input boxes cannot use the common style of our `<Input />` component in design, and require an `<Input native/>` component that can be adjusted more easily.

Reply:
Maybe we can extract logic from `<Input>` as a new component like `<RawInput />`, and use `<RawInput />` inside `<Input>`?